app/src/main/res/values/dimens.xml

```diff
@@ -227,6 +227,10 @@
     <dimen name="top_sites_card_radius">8dp</dimen>
     <dimen name="top_sites_card_stroke_width">1dp</dimen>
 
+    <!-- Recent tab  -->
+    <dimen name="recent_tabs_icon_elevation">1dp</dimen>
```

Comment:
Did we need this elevation? This was `0dp` for the favicon in top sites. See `top_sites_favicon_elevation`.

Reply:
I got this value from `recentBookmarkFaviconShape`.  You're correct that it should be 0 dp.  As discussed, I'll update `recentBookmarkFaviconShape` to 0 dp as well.